Units/mtable-simple-with-table-extending.d/args.ctags

```diff
@@ -6,6 +6,8 @@
 --kinddef-X=v,var,variables
 --kinddef-X=f,fun,functions
 
+--kinds-X=+{var}
+
```

Comment:
Can I ask you to make a separate test case?
This test case will be important when I rewrite the code for parser initialization.
I think this should go to Tmain.

Reply:
Yes I'll make it a separate test case and update the PR.